DOC-3132: Expanding selection to word didn't work inside inline editing host elements. (#3626)
tinymce/tinymce-docs: modules/ROOT/pages/7.7.0-release-notes.adoc

{productname} {release-version} resolves this issue by replacing the `title` attribute with the `aria-label` attribute for toolbar groupings, preventing the default browser tooltip from appearing.

> === Expanding selection to word didn't work inside inline editing host elements.

Reviewer: ```suggestion
=== Expanding selection to a word didn't work inside inline editing host elements.
```

// #TINY-11304

> Previously, an issue occurred where the selection did not expand as expected when "expand to word" action was triggered within an inline edit host element inside a non-editable parent.

Reviewer 2: Maybe we point to the API here:

```suggestion
Previously, an issue occurred where the selection did not expand as expected when using the `editor.selection.expand({ type: 'word' })` API when the selection was within an inline edit host element inside a non-editable parent.
```

> Previously, an issue occurred where the selection would not expand as expected when expand to word was called within an inline edit host element inside a non-editable parent.
> 
> This has been resolved in {productname} {release-version}, ensuring that the expanded selection now remains within the boundaries of the editable host element.

Reviewer: ```suggestion
Previously, an issue occurred where the selection did not expand as expected when "expand to word" action was triggered within an inline edit host element inside a non-editable parent.

This has been resolved in {productname} {release-version}, ensuring that the expanded selection now remains within the boundaries of the editable host element.
```
